test/new-e2e/pkg/utils/infra/stack_manager.go

```diff
@@ -261,7 +258,13 @@ func buildWorkspace(ctx context.Context, profile runner.Profile, stackName strin
 		return nil, fmt.Errorf("unable to create temporary folder at: %s, err: %w", workspaceStackDir, err)
 	}
 
-	return auto.NewLocalWorkspace(ctx, auto.Project(project), auto.Program(runFunc), auto.WorkDir(workspaceStackDir))
+	fmt.Printf("Creating workspace for stack: %s at %s", stackName, workspaceStackDir)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
What about passing `t` context and using `t.Log` instead ?

Reply:
I don't think that `stackManager` should be test-aware, but with the changes done to pass a logger, we can implement that.